linux/net/rina/serdes.c

```diff
@@ -1065,3 +1101,82 @@ struct pdu * pdu_deserialize_ni(const struct serdes * instance,
                                 struct crypto_blkcipher * blkcipher)
 { return pdu_deserialize_gfp(GFP_ATOMIC, instance, pdu, dup_conf, blkcipher); }
 EXPORT_SYMBOL(pdu_deserialize_ni);
+
+int serdes_pci_size(pdu_type_t type, struct dt_cons * c) {
```

Comment:
Do you really want to remove the possibility to know just how much a pci weight?